docs/Documentos/eap.md

```diff
@@ -1,11 +1,3 @@
----
-layout: default
-title: Estrutura Analítica do Projeto
-nav_order: 1
-parent: Documentos
-
----
-
 # Estrutura Analítica do Projeto (EAP)
```

Comment:
Não sei se ocorreu só comigo, mas ao tentar acessar a página "Documentos" pelo docsify, deu erro 404 - Not found

Reply:
Obrigado, já vou corrigir

Reply:
Na verdade vc e a Aline já terão corrigido no PR #23 de vcs !